### DISCUSSION TITLE
help with 3d models

### DISCUSSION THREAD

**Discussion by hernan74** [CariusLars/ar_flutter_plugin#31] [Q&A] [answered]
Help for 3d images in glb format read from the web.
I am generating my models using blender that generates in glb 2.0.
When wanting to load the model, it generates the following error:
` Unable to load Renderable registryId='https://github.com/hernan74/ejemplo_realidad_aumentada/blob/main/assets/adobe_ejemplo.glb'
E/ModelRenderable(17523): java.util.concurrent.CompletionException: java.lang.RuntimeException: Failed to import '/data/user/0/com.example.modelo_base/cache/1631623018293-0/model_source.glb'
E/ModelRenderable(17523): 	at com.google.ar.sceneform.utilities.SceneformBufferUtils.inputStreamToByteBuffer(SourceFile:48)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.rendering.LoadRenderableFromSfbTask.lambda$downloadAndProcessRenderable$0$LoadRenderableFromSfbTask(LoadRenderableFromSfbTask.java:118)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.rendering.-$$Lambda$LoadRenderableFromSfbTask$A8YYsoT3V_A--F80fBJ_ZLNSnhc.get(Unknown Source:4)
E/ModelRenderable(17523): 	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1627)
E/ModelRenderable(17523): 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
E/ModelRenderable(17523): 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
E/ModelRenderable(17523): 	at java.lang.Thread.run(Thread.java:919)
E/ModelRenderable(17523): Caused by: java.lang.RuntimeException: Failed to import '/data/user/0/com.example.modelo_base/cache/1631623018293-0/model_source.glb'
E/ModelRenderable(17523): 	at com.google.ar.sceneform.assets.Converter.convertModel(Native Method)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.assets.Converter.a(Unknown Source:129)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.assets.RenderableSource.call(Unknown Source:226)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.assets.RenderableSource.call(Unknown Source:0)
E/ModelRenderable(17523): 	at com.google.ar.sceneform.utilities.SceneformBufferUtils.inputStreamToByteBuffer(SourceFile:42)
E/ModelRenderable(17523): 	... 6 more
I/flutter (17523): [Unable to load renderablehttps://github.com/hernan74/ejemplo_realidad_aumentada/blob/main/assets/adobe_ejemplo.glb]
`

**Comment by CariusLars** [comment, marked as answer]
try with the "raw" flag turned on in github: https://github.com/hernan74/ejemplo_realidad_aumentada/blob/main/assets/adobe_ejemplo.glb?raw=true

**Reply by hernan74** [reply]
Thank you very much it worked for me
